app/components/UserValidator/index.tsx

```diff
@@ -145,4 +221,8 @@ const Validator = (props: Props) => {
   );
 };
 
-export default Validator;
+const mapDispatchToProps = {
+  addToast,
+};
+
+export default connect(null, mapDispatchToProps)(Validator);
```

Comment:
Preferably add this together with the rest in `app/routes/userValidator/ValidatorRoute.tsx`.